src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,120 @@
+package main.java.duke;
+
+import java.util.ArrayList;
+
+import main.java.duke.tasks.Task;
+
+/**
+ * Represents the current the task list.
+ */
+public class TaskList {
+    private ArrayList<Task> taskList;
```

Comment:
It would be better to name the variable "tasks" in this case as we would want to differentiate between single and multi-valued variable!!